ts/annotations/annotations.src.ts

```diff
@@ -169,9 +169,11 @@ declare global {
         interface AnnotationsTypeOptions {
             background?: AnnotationsShapeOptions;
             height?: number;
+            labelOptions?: Partial<AnnotationsShapeOptions>;
             line?: AnnotationsShapeOptions;
             point: AnnotationMockPointOptionsObject;
             points?: Array<AnnotationsTypePointsOptions>;
+            shapes?: AnnotationsShapeOptions;
```

Comment:
Is this type used later to generate ts definitions for users? If yes, then we can not set it here. We don't have `shapes` and `labelOptions` options available for any annotation type.